trunner/harness/plo.py

```diff
@@ -75,7 +76,12 @@ def enter_bootloader(self):
         try:
             self.dut.expect_exact("Waiting for input", timeout=3)
         except (pexpect.TIMEOUT, pexpect.EOF) as e:
-            raise PloError("Failed to go into bootloader mode", output=self.dut.before) from e
+            raise PloError(
```

Comment:
styling changes are not needed, we have a soft limit of 120 chars per line for python - please configure your editor correctly :exclamation: 

(eg. https://github.com/phoenix-rtos/phoenix-rtos-tests/blob/master/.github/workflows/runner-ci.yaml#L43)

Reply:
Funny thing that I have this configuration and it formats this 
![image](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/49757239/1db1cb73-2495-42a2-997e-aea09c329c70)